systers_portal/templates/meetup/meetup.html

```diff
@@ -1,6 +1,7 @@
 {% extends "meetup/base.html" %}
 
 {% block meetup_location_page_content %}
+  <script type="text/javascript" src="http://w.sharethis.com/button/buttons.js"></script>
```

Comment:
Instead of having it here, can we have move it to a common place, as then we can allow meetup location, meetup and others to have these share buttons?